app/views/photos/index.html.haml

```diff
@@ -0,0 +1,3 @@
+No photos added yet
+%br
+%a(href='#')Add a photo
```

Comment:
Prefer the hash attributes syntax (%tag{ lang: 'en' }) over HTML attributes syntax (%tag(lang=en))